src/stylesheets/screens/survey-list.scss

```diff
@@ -8,10 +11,6 @@
     border-radius: rem(12px);
   }
 
-  &__information {
-    display: flex;
-  }
-
   &__title {
     margin-top: rem(20px);
```

Comment:
Should we change this to `rem(32px)`? 
![Screen Shot 2563-11-18 at 11 27 41](https://user-images.githubusercontent.com/1772999/99483676-4c2dd180-2991-11eb-9301-083a81a6f838.png)

Reply:
Fixed in [1bbdeea](https://github.com/rafayet-monon/surveyor/pull/17/commits/1bbdeeae86507eb1b8ecd22536d5bd0d7a579e3c)